frontend/src/routes/Profile/Components/EditPersonalInfoContainer.tsx

```diff
@@ -117,11 +117,12 @@ const EditPersonalInfoContainer = () => {
   }
 
   const EditProfileImage = () => {
+    // TODO doesn't work
```

Comment:
I realize this hasn't been working since 2021, so it's probably something we want to explore why